nbhttp/websocket/conn.go

```diff
@@ -139,10 +139,11 @@ func (c *Conn) WriteMessage(messageType MessageType, data []byte) error {
 		cw := compressWriter(w, c.compressionLevel)
 		_, err := cw.Write(data)
 		if err != nil {
-			return err
+			compress = false
```

Comment:
Do not return err, but continue to send data without compression, then we do not need to judge the return value of WriteMessage, else we don't know whether the connection has been closed by the framework.

Reply:
this is merged in from later commits